tests/src/texture/tex1Dfetch_linear.cpp

```diff
@@ -0,0 +1,106 @@
+/* HIT_START
```

Comment:
Can't this test be combined into the above one i.e. into hipBindTexObj1D.cpp? Both seem to be working on 1d textures with fetch in linear mode.